library/std/src/io/stdio.rs

```diff
@@ -1016,6 +1017,132 @@ where
     }
 }
 
+mod terminal_sealed {
+    #[unstable(feature = "sealed", issue = "none")]
+    pub trait Sealed {}
+}
```

Comment:
We already have `std::sealed::Sealed` for this. No need to create new a `Sealed` trait.

Reply:
I'd originally created this separate Sealed to allow for the possibility of a blanket impl, but now that this doesn't have a blanket impl anymore it can just use the standard `Sealed`. Will change.